src/components/pickers/DatePicker.js

```diff
@@ -19,60 +20,38 @@ export default {
       currentMonth: null,
       currentYear: null,
       isSelected: false,
-      isReversing: false
+      isReversing: false,
+      narrowDays: []
     }
   },
 
   props: {
+    locale: {
+      type: String,
+      default: 'en-us'
+    },
     dateFormat: {
       type: Function,
       default: defaultDateFormat
     },
     titleDateFormat: {
-      type: Function,
-      default: ({ day, dayName, month, monthName, landscape }) => {
-        return `${dayName.substr(0, 3)},${landscape ? '<br>' : ''} ${monthName.substr(0, 3)} ${day}`
-      }
+      type: Object,
+      default: () => ({ weekday: 'short', month: 'short', day: 'numeric' })
     },
     headerDateFormat: {
-      type: Function,
-      default: ({ month, monthName, year }) => `${monthName} ${year}`
-    },
-    days: {
-      type: Array,
-      default: () => ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']
-    },
-    shortDays: {
-      type: [Function, Array],
-      default: (day) => day && day.substr(0, 1)
+      type: Object,
+      default: () => ({ month: 'long', year: 'numeric' })
     },
     formattedValue: {
       required: false
     },
-    months: {
-      type: Array,
-      default: () => [
-        'January',
-        'February',
-        'March',
-        'April',
-        'May',
-        'June',
-        'July',
-        'August',
-        'September',
-        'October',
-        'November',
-        'December'
-      ]
-    },
     allowedDates: {
       type: [Array, Object, Function],
       default: () => (null)
     },
     firstDayOfWeek: {
-      type: String,
-      default: 'Sunday'
+      type: Number,
```

Comment:
Should make this [Number, String] so the user doesn't have to bind a prop in order to change this